File: Presentations/CMSC 21 - Arrays and Strings (Yves)/codes/1. green folder/find-green-folder.c
#include <stdio.h>

void main() {

    // for the purpose of example, declare variables
    int box = 1;
    int bottle = 2;
    int green_folders = 3;

    // this contains all the items
    int container[7] = {0, box, 0, 0, bottle, green_folders, 0};
    
    // acquired item will go here
    int acquired_item;

    // go through each of the items
    for (int i=0; i<7; i++) {

        // check if current item is the green folders
        if (container[i] == green_folders) {

            // if it is, put value to variable 'acquired_item'
            acquired_item = container[i];

            // stop the current iteration
            break;
        }
    }

    
    // check if you got the green folders
    if (acquired_item == green_folders) {
        printf("*Tigershark grabs item*\nSuccess!");
    } else {
        printf("*Tigershark is confused*\nItem not found in container...");
    }

}

File: Presentations/CMSC 12 - Strings, Lists, Dictionaries (Yves)/codes/1. green folder/find-green-folder.py
def main():
    # for the purpose of example, declare variables
    box = 1
    bottle = 2
    green_folders = 3
    empty = 0

    # this contains all the items
    container = [0, box, 0, 0, bottle, green_folders, 0]
    
    # acquired item will go here
    acquired_item = None # None = wala pa po

    # iterate through indices based on the length of the container
    for i in range(len(container)):
        
        # check if current item is the green folders
        if container[i] == green_folders:
            
            # access value in specific box
            acquired_item = container[i]
            
            # stop the loop
            break
    
    # print notification of green folders
    if acquired_item == green_folders:
        print("*Tigershark grabs item*\nSuccess!")
    else:
        print("*Tigershark is confused*\nItem not found in container...")

# if not a module
if __name__ == "__main__":
    main()
